orchestration/dagster_orchestration/hca_orchestration/sensors.py

```diff
@@ -29,6 +29,7 @@ def generate_run_request(self, workflow: ExtendedArgoWorkflow) -> RunRequest:
         inflated_workflow = workflow.inflate()
 
         dataset_name = inflated_workflow.params_dict()['data-repo-name'].removeprefix("datarepo_")
+        argo_id = inflated_workflow.metadata.uid
```

Comment:
⛏️ I think we should call this `argo_workflow_id`, as that's the entity to which the identifier is attached.